docs/DeveloperGuide.md

```diff
@@ -2,12 +2,111 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+We first give acknowledgement to Module Coordinator, Dr Akshay Narayan, and Teaching Assistant, Aditi Chadha for guidance and supervision over the KaChinnnng project.
+
+We utilised the following resources to aid us in the development of KaChinnnng:
+-  [tp project](https://github.com/nus-cs2113-AY2324S1/tp) of the nus-cs2113-AY2324S1 organisation
+-  [addressbook-level2](https://github.com/se-edu/addressbook-level2) & [addressbook-level3](https://github.com/se-edu/addressbook-level3) project by SE-EDU largely inspired the format and coding style of KaChinnnng.
+
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+Below are the design and implementation details of KaChinnnng.
+
+### Design
+
+The architectural diagram of KaChinnnng is as follows:
+
+![ArchitectureDiagram.png](https://github.com/AY2324S1-CS2113-T18-3/tp/blob/master/images/ArchitectureDiagram.png?raw=true)
```

Comment:
The architecture diagram is a little complicated as it has a lot going on and each component has that figure in it, which is also distracting. Also, is the **Run** component supposed to have the **()**?